packages/fxa-auth-server/test/scripts/bulk-mailer/send-email-batches.js

```diff
@@ -82,16 +81,15 @@ describe('send-email-batches', () => {
     assert.strictEqual(sendEmailBatchSpy.args[1][1], sender);
   });
 
-  it(
-    'uses a delay between batches',
-    async () => {
-      await retry(async () => {
+  it('uses a delay between batches', async () => {
+    await retry(
+      async () => {
         assert.isAbove(totalTimeMS, 100);
-      });
-    },
-    {
-      retries: 10,
-      minTimeout: 20,
-    }
-  );
+      },
+      {
+        retries: 10,
+        minTimeout: 20,
+      }
+    );
+  }).timeout(8000);
```

Comment:
When I wrote this test, I apparently put the options to `retry` in the wrong location, as they were actually being passed to the `it` function. They are now passed properly to the `retry` function.